app/internal/languages.py

```diff
@@ -0,0 +1,85 @@
+import glob
+import json
+import pathlib
+from typing import Dict, Union
+
+from app import config
+
+LANGUAGE_FILES_PATH = "app/languages/*.json"
+LANGUAGE_FILES_PATH_TEST = "../app/languages/*.json"
+
+translations_dict = {}
```

Comment:
Prefer not to use a global state

Reply:
Is there a different method you prefer or should I perform this whole process each time a route is accessed?

Reply:
Resolved with @lru_cache() (as a final fix for current version).